src/main/java/io/kestra/plugin/microsoft365/sharepoint/Create.java

```diff
@@ -0,0 +1,180 @@
+
+import java.net.URI;
+import java.util.Collections;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Plugin(
+    examples = {
+        @io.kestra.core.models.annotations.Example(
+            title = "Create a new file in a SharePoint document library.",
+            code = {
+                "siteId: \"your-site-id\"",
+                "driveId: \"your-drive-id\"",
+                "parentId: \"your-parent-folder-id\"",
+                "filename: \"example.txt\"",
+                "content: \"File content here\""
+            }
+        )
+    }
+)
+@Schema(
+    title = "Create a new file or folder in a SharePoint document library.",
+    description = "This task allows you to create a new file or folder in a SharePoint document library."
+)
+public class Create extends Task implements RunnableTask<Create.Output> {
+    @Schema(
+        title = "The SharePoint site ID.",
+        description = "The unique identifier of the SharePoint site."
+    )
+    @PluginProperty(dynamic = true)
+    private Property<String> siteId;
+
+    @Schema(
+        title = "The SharePoint drive ID.",
+        description = "The unique identifier of the SharePoint document library (drive)."
+    )
+    @PluginProperty(dynamic = true)
+    private Property<String> driveId;
+
+    @Schema(
+        title = "The parent item ID.",
+        description = "The unique identifier of the parent folder where the new item will be created."
+    )
+    @PluginProperty(dynamic = true)
+    private Property<String> parentId;
+
+    @Schema(
+        title = "The filename or folder name.",
+        description = "The name of the file or folder to create."
+    )
+    @PluginProperty(dynamic = true)
+    private Property<String> filename;
+
+    @Schema(
+        title = "The content of the file.",
+        description = "The content to be written to the new file. If not provided, an empty folder will be created."
+    )
+    @PluginProperty(dynamic = true)
+    private Property<String> content;
+
+    @Override
+    public Create.Output run(RunContext runContext) throws Exception {
+        Logger logger = runContext.logger();
+        
+        String renderedSiteId = runContext.render(siteId).as(String.class).orElse(null);
+        String renderedDriveId = runContext.render(driveId).as(String.class).orElse(null);
+        String renderedParentId = runContext.render(parentId).as(String.class).orElse(null);
+        String renderedFilename = runContext.render(filename).as(String.class).orElse(null);
+        String renderedContent = runContext.render(content).as(String.class).orElse(null);
+        
+        logger.debug("Creating file/folder '{}' in SharePoint site '{}', drive '{}'", 
+            renderedFilename, renderedSiteId, renderedDriveId);
+        
+        try {
+            com.microsoft.graph.requests.GraphServiceClient<?> graphClient = GraphClientProvider.getClient();
```

Comment:
Just add the import:

```suggestion
            GraphServiceClient<?> graphClient = GraphClientProvider.getClient();
```